File: src/main/java/injea/knwremodel/Mail/MailDTO.java
package injea.knwremodel.Mail;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

public class MailDTO {
    @Getter
    @AllArgsConstructor
    @NoArgsConstructor
    public static class send{
        private String mail;
    }

    @Getter
    @AllArgsConstructor
    @NoArgsConstructor
    public static class confirmNumber{
        private String enteredNumber;
    }
}

File: src/main/kotlin/injea/knwremodel/Mail/MailDTO.kt
package injea.knwremodel.Mail

class MailDTO {
    class send {
        var mail: String? = null
            private set

        constructor(mail: String?) {
            this.mail = mail
        }

        constructor()
    }

    class confirmNumber {
        var enteredNumber: String? = null
            private set

        constructor(enteredNumber: String?) {
            this.enteredNumber = enteredNumber
        }

        constructor()
    }
}

